superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -52,4 +52,10 @@ const Tabs = styled(BaseTabs)`
   }
 `;
 
+const StyledTabPane = styled(AntdTabs.TabPane)``;
+
+const Tabs = Object.assign(StyledTabs, {
+  TabPane: StyledTabPane,
+});
```

Comment:
This Object.assign technique lets you individually style all the compound components the AntD API offers up. Should make things easier for devs who know AntD.

Reply:
We should add a README to `src/common/components` with info on best practices for extending antd components with styles.

Reply:
Good idea... I have a fair amount of component documentation that might be good to have in there. It may be subject to various opinions, so I'll do it in another PR.

Reply:
So instead of what I'm currently doing in the database modal:
```
import Tabs from 'src/common/components/Tabs';
import { Tabs as BaseTabs } from 'src/common/components';

const { TabPane } = BaseTabs;
```

I can just use this?
```
import Tabs from 'src/common/components/Tabs';

const { TabPane } = Tabs;
```

Reply:
@riahk I _believe_ you could, yes. I already made changes in this PR to do the `<Tabs.TabPane>` which seems to be the convention for AntD. Let me know if you object, before I merge this.